AAI/AAIConnectProfile.md

```diff
@@ -222,7 +223,7 @@ the Broker.
         2.  Pragma: no-cache
         
 5.  MUST provide protection against Client attacks as outlined in
-    [RFC 6819](https://tools.ietf.org/html/rfc6819).
+    [RFC 6819](https://tools.ietf.org/html/rfc6819) section-4.1.
```

Comment:
section-4.1 is too narrow, and there are several other sections of the RFC that clients need to adhere to. Section 5 is another example, but still doesn't cover it. I think it is better to just leave this pointing to the whole spec. "do all of it related to a client".